HowTo install Linux (Ubuntu) tools for C++ programming with Eclipse IDE/readme.md

```diff
@@ -83,12 +84,12 @@ Unzip **SDK**:
 
 Download the needed SDK from the Phoenix Contact webside.
 
-[LINK](http://www.phoenixcontact.com/qr/2404267/softw)
+[LINK](https://www.phoenixcontact.com/online/portal/pi?uri=pxc-oc-itemdetail:pid=2404267&library=piru&tab=5&requestType=qr&productId=2404267#softw)
 
 Navigate to the folder where your SDK is located.
 
 ```sh
-unzip SDK_Linux64_2020.0.zip
+unzip PLCnCLI_SDK_2021.0.5_LTS_Linux_AXC_F_2152.tar.gz
```

Comment:
SDK is downloaded separately - so we can download only required toolchain `pxc-glibc-x86_64-axcf2152-image-sdk-cortexa9t2hf-neon-axcf2152-toolchain-2021.9.sh`.